Crm/Company.php

```diff
@@ -719,6 +740,21 @@ public function toArrayForApi()
         if ($this->getWebsite()) {
             $return['website'] = $this->getWebsite();
         }
+        if ($this->getBic()) {
+            $return['bic'] = $this->getBic();
```

Comment:
This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php